### PR TITLE
Fix some type errors

### PR DESCRIPTION
**Pull request by cktang88**
Progress on #99

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,7 +119,7 @@
 		"@types/react-router-dom": "^4.3.3",
 		"@types/react-select": "^2.0.17",
 		"@types/react-virtualized": "^9.21.1",
-		"@types/styled-components": "^4.1.15",
+		"@types/styled-components": "4.1.8",
```
Comment (bencooper222) [inline package.json]: Why the pinning?

Comment (cktang88) [reply package.json]: It fixes the issue with `@types/react-native`, see issue99

Comment (bencooper222) [reply package.json]: Gotcha. Would the postinstall rm for the bad types solution be better for us?

Comment (cktang88) [reply package.json]: I don't see any disadvantage of just pinning version, everything still works

Comment (bencooper222) [reply package.json]: Sounds good

**Pull request merged**